src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> inventory;
```

Comment:
If this `inventory` map is intended to represent a shared state across the application (which is common for a storage class), it should be declared as `static`. This way, it ensures there is only one instance of the inventory throughout the application, which is the typical behavior expected from a storage component.